docs/examples/webpack-entry-point.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Webpack entry point
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This example is the same as the [basic example](basic), but using the webpack entry point to import the library styles.
```

Comment:
but using webpack entry point in order to import the library (you can use the _entry_ section of your _webpack.config.js_ and add there the path to the library css file instead of having to add it in your main HTML / JS file in your app.